config/routes.rb

```diff
@@ -16,6 +16,13 @@
     end
     resources :products, only: [:show]
     resources :sessions, only: [:new, :create, :destroy]
-    resources :orders, only: [:new, :create]
+    resources :orders, only: [:new, :create, :index] do
+      collection do
+        put "update_status/:id", to: "orders#update_status", as: "update_status"
+      end
+      member do
+        resources :order_items, only: [:index]
+      end
```

Comment:
dùng nested resources rồi thì ko cần đưa vào member nữa e nha
```suggestion
        resources :order_items, only: [:index]
```